joy_params:
  joy_scale:
  right_scale_trigger:
  left_scale_trigger:

